### PR TITLE
sGAR ~ 2024.02.20 ~ FAIR 412 --> EGSY airport code duplicates removed

### PR DESCRIPTION
**Pull request by ame-test**
**What changes does this PR bring?**
- Remove duplicate EGSY airport code, prevent error when EGSY is used.
- Updates the logs so that error logs output errors and success logs occur on success of resolve --> making it easier to debug the app.

FAIR 412 requests:
* https://gitlab.digital.homeoffice.gov.uk/egar/e2e-test/-/merge_requests/232
* https://github.com/UKHomeOffice/egar-public-site-ui/pull/445

**Definition of Done**
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.
- [ ] Have you built the code locally and confirmed its working?
- [ ] Is the build confirmed to be passing on CI?
- [ ] Have you added enough relevant unit tests for your change?
- [ ] Have you added enough relevant E2E tests for your change?
- [ ] Have you updated any relevant documentation as part of this change?
- [ ] Has this change been tested against the Acceptance Criteria?
- [ ] Have you considered how this will be deployed in SIT/Staging/Production?

### CODE REVIEW

**Comment by ame-test** [comment]
Pass test
Screenshot 2024-02-21 at 10 03 14
Screenshot 2024-02-21 at 10 03 27

**Review by sjose012 — APPROVED** [review]
Approved, just confirm with business before merging.

File: src/common/utils/airport_codes.json [hunk 1]
```diff
@@ -3368,13 +3368,6 @@
     "crownDependency": false,
     "label": "RNAS Culdrose (United Kingdom) (EGDR)"
   },
-  {
```
[the remark this answers is not in the document]
Comment (ho-henry-senior) [reply src/common/utils/airport_codes.json:3371]: Hi @ame-test did EGDX need to be removed as well as EGSY?

[the remark this answers is not in the document]
Comment (ame-test) [reply src/common/utils/airport_codes.json]: The EGDX has been added back intot o be inloine with Sonali email:
```
For confirmation only – the following two locations should appear on sGAR reference data:
* St Athan (EGSY)
* MOD St Athan (EGDX)
```

[the remark this answers is not in the document]
Comment (ho-henry-senior) [reply src/common/utils/airport_codes.json]: thanks Al

**Pull request merged**